app/models/cached_uphold_tip.rb

```diff
@@ -0,0 +1,18 @@
+class CachedUpholdTip < ApplicationRecord
+  belongs_to :uphold_connection_for_channel
+
+  def channel
```

Comment:
May you change the method name? Maybe publication title

Reply:
https://github.com/brave-intl/publishers/pull/2682/commits/ce7f6316dae0c7c5d5202f7a44f8c0c6da1e9cd6